src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/controller/v1_0/FileHistoryController.java

```diff
@@ -15,7 +15,6 @@
 import org.springframework.web.bind.annotation.RestController;
 
 import ca.bc.gov.open.jag.tco.oracledataapi.model.FileHistory;
```

Comment:
I'm a bit confused about what is going on here.  This would definitely be a breaking change, wouldn't it?  Can we have a new controller AuditLogEntryController, new methods? If there is a good reason to change the fileHistoryController thats ok, but otherwise we could just have AuditLogEntryController which is always pointing to Oracle, and fileHistoryController always H2.  Once we do the switch we are not going back to H2.

Reply:
Yes, this would be a breaking change if we merge this in as it is right now. However, I'm also working on the second part of this to refactoring .net apps that are affected by this change (TCVP-2077). That work is on a branch which is branched off of this one so once the second part of this work will be merged into this one (TCVP-2075), this will no longer a breaking change.

I don't see any value in adding extra code to trying to make it work with H2 at his point since our goal is completely switching to use ORDS. I created this PR without completing the fixing the breaking change part just because to make it easier to review.

Reply:
Yeah, thanks for keeping the PRs shorter - it makes things a lot easier. I marked this PR as a Breaking Change so to remember not to merge it in until it's complete with your other work. When you make your other PRs, set them to merge into this branch instead of main so we can see the differences, then when all changes are ready, this branch can be merged all at once.

Reply:
Yes, this was the approach that I was thinking exactly. Will do that.